.github/workflows/docker-verify.yaml

```diff
@@ -61,15 +61,20 @@ jobs:
             "quantum-serverless-notebook:latest-py38",
             "quantum-serverless-notebook:latest-py39",
             "quantum-serverless-notebook:latest-py310",
+            "quantum-serverless-notebook:latest-py311",
             "quantum-serverless-ray-node:latest-py38",
             "quantum-serverless-ray-node:latest-py39",
             "quantum-serverless-ray-node:latest-py310",
+            "quantum-serverless-ray-node:latest-py311",            
             "quantum-repository-server:latest",
           ]
         include:
           - image: "quantum-serverless-gateway:latest"
             dockerfile: "./infrastructure/docker/Dockerfile-gateway"
             pyversion: "3.9"
+          - image: "quantum-serverless-gateway:latest-py311"
+            dockerfile: "./infrastructure/docker/Dockerfile-gateway"
+            pyversion: "3.11"
```

Comment:
The `gateway` and `repository` ones can be removed because they don't take into account the python version right now. They only use python `3.9` by now.

Reply:
Yeah, but those `Dockerfiles` are not taking into account the `pyversion` right now. They are being built directly with `3.9`:

https://github.com/Qiskit-Extensions/quantum-serverless/blob/main/infrastructure/docker/Dockerfile-gateway#L1

Reply:
well, that's not ideal...